content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,212 @@
+
+- **None**: This setting requires you to configure OIDC manually in the Kubernetes pack.
+
+### Manually Configure OIDC
+
+You only need to configure OIDC manually if you change the **Identify Provider** setting to **None**. The basic method to enable OIDC can be used for all cloud services except Amazon EKS. 
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Basic OIDC Setup" key="Other Clouds">
+
+
+To configure OIDC manually for clusters managed by most cloud providers, follow these steps: 
+
+<br />
+
+1. Copy ``oidc-`` configuration lines in the example and add them to the Kubernetes pack under the ``extraArgs`` parameter section. Enter your third-party provider details in quotes. <br /><br />
+
+  ```
+  kubeadmconfig:
+    apiServer:
+      extraArgs:
+      oidc-issuer-url: "provider URL"
+      oidc-client-id: "client-id"
+      oidc-groups-claim: "groups"
+      oidc-username-claim: "email"
+  ```
+
+2. Under the ``clientConfig`` parameter section of Kubernetes pack, uncomment the ``oidc-`` configuration lines, and enter your provider details in quotes. The provider URL and client-id must be the same in the ``extraArgs`` and ``clientConfig`` parameter sections. <br /><br />
+
+  ```
+    clientConfig:
+      oidc-issuer-url: "provider URL"
+      oidc-client-id: "client-id"
+      oidc-client-secret: client-secret-value
+      oidc-extra-scope: profile,email,openid
+    ```
+
+<br />
+
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="AWS EKS" key="AWS EKS">
+
+To enable OIDC manually for EKS clusters, follow these steps: 
+
+<br />
+
+1. In the Kubernetes pack, uncomment the lines in the ``oidcIdentityProvider`` parameter section of the Kubernetes pack, and enter your third-party provider details.
+<br />
+
+  ```
+    oidcIdentityProvider:
+      identityProviderConfigName: 'Spectro-docs'     # The name of the OIDC provider configuration
+      issuerUrl: 'issuer-url'       # The URL of the OpenID identity provider
+      clientId: 'user-client-id-from-Palette'           # The ID for the client application that makes authentication requests to the OpenID identity provider
+      usernameClaim: "email"                     # The JSON Web Token (JWT) claim to use as the username
+      usernamePrefix: "-"                        # The prefix that is prepended to username claims to prevent clashes with existing names
+      groupsClaim: "groups"                      # The JWT claim that the provider uses to return your groups
+      groupsPrefix: ""                          # The prefix that is prepended to group claims to prevent clashes with existing names
+      requiredClaims:                            # The key value pairs that describe required claims in the identity token
```

Comment:
```suggestion
    oidcIdentityProvider:
      identityProviderConfigName: "Spectro-docs" 
      issuerUrl: "issuer-url"    
      usernameClaim: "email"     
      usernamePrefix: "-"      
      groupsClaim: "groups"      
      groupsPrefix: ""                          
      requiredClaims:  ""                          
```
I removed the comments to improve the readability and make it easier for people to copy-paste the code snippet.